pkg/status/status.go

```diff
@@ -36,10 +36,14 @@ func (o UpdateStatusOptions) UpdateBackupStatusFromFile() error {
 	if err != nil {
 		return err
 	}
+
 	var backupErr error
-	if backupOutput.HostBackupStats.Error != "" {
-		backupErr = fmt.Errorf(backupOutput.HostBackupStats.Error)
+	for _, hostStats := range backupOutput.HostBackupStats {
+		if hostStats.Error != "" {
+			backupErr = errors.NewAggregate([]error{backupErr, fmt.Errorf(hostStats.Error)})
```

Comment:
This is not right. We should collect all the errors in a `[]error`, then call 

`errors.NewAggregators(errs)`